### PR TITLE
Catapult launch

### PR DESCRIPTION
**Pull request by Thebloodshadow**
Added catapult launch (Change timer for the right time).
Try and test the motorbit at the end of the code (May be easier to do with than homemade fonction)

### CODE REVIEW

**Comment by Rheydskey** [comment]
It's seem to be good to merge @Vortezz

**Comment by Vortezz** [comment]
Some things may be modified, like the sseconds variable...

File: main.py [hunk 1]
```diff
@@ -4,7 +4,8 @@
 # NOTICE : Some value has to be #
 #   edited before utilisation   #
 #################################
-
+seconds_launch = 0
+sseconds = 0
```
Comment (Rheydskey) [inline main.py]: Why double s

**Review by Rheydskey — APPROVED** [review]

**Comment by Vortezz** [comment]
I think it will be good soon @Thebloodshadow

**Pull request closed without merging**